vignettes/SingleR.Rmd

```diff
@@ -1,7 +1,7 @@
 ---
 title: Using SingleR to annotate single-cell RNA-seq data
-author: Aaron Lun
-date: "Revised: 17 October 2018"
+author: Aaron Lun, Jared Andrews
```

Comment:
Use the `- name:` set up shown [here](https://github.com/MarioniLab/simpleSingleCell/blob/master/vignettes/batch.Rmd). You can also put your affiliation if you so please.